src/Corax/Indexing/IndexWriter.Testing.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Corax.Utils;
+using Sparrow;
+using Sparrow.Compression;
+using Sparrow.Server;
+using Voron.Data.Containers;
+using Voron.Data.PostingLists;
+
+namespace Corax.Indexing;
+
+public partial class IndexWriter
+{
+    internal TestingInterface CreateTestingInterface() => new(this);
```

Comment:
For consistency and our conventions can we make the following changes:
1. `TestingInterface` -> `TestingStuff`
2. `CreateTestingInterface()` -> `ForTestingPurposes`
?